packages/utils/scripts/compare_resource_counts_per_file.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+## This script pulls the per resource count reference file from s3 and compares it to the file passed in as an argument
+
+# Check if one argument is passed
+if [ "$#" -ne 1 ]; then
+  echo "Usage: $0 <file1>"
+  exit 1
+fi
+
+# Assign argument to a variable for better readability
+FILE1=$1
+
+# Fixed S3 path for file2
+S3_FILE2="s3://fhir-resource-count/develop-per-file-resource-counts.json"
+
+# Temporary file to store the downloaded S3 file
+TMP_FILE2=$(mktemp)
+
+# Download file2 from S3, suppressing stdout but keeping stderr
+if aws s3 cp "$S3_FILE2" "$TMP_FILE2" > /dev/null; then
+  echo "File downloaded successfully, running jsondiffpatch."
+else
+  echo "Failed to download $S3_FILE2"
+  exit 1
+fi
+
+# Run jsondiffpatch
+jsondiffpatch "$FILE1" "$TMP_FILE2"
```

Comment:
Does this needs to be installed? Add it to the README if it can't be installed automatically by adding it to `devDependencies` on `package.json`